controllers/datasciencecluster/datasciencecluster_controller.go

```diff
@@ -384,8 +384,29 @@ var configMapPredicates = predicate.Funcs{
 		if e.ObjectNew.GetName() == "inferenceservice-config" && (namespace == "redhat-ods-applications" || namespace == "opendatahub") { //nolint:goconst
 			return false
 		}
+		// Trigger reconcile function when uninstall configmap has this label
```

Comment:
this seems to be unrelated right ?

Reply:
i have a feeling.......... this is not the code on my local branch :D 
let me push it and see what the changes :D